app/assets/javascripts/test/model/binary/layers/wk_layer.spec.js

```diff
@@ -115,19 +101,18 @@ test.serial("requestFromStore: Token Handling should re-request a token when it'
 test.serial("requestFromStore: Request Handling: should pass the correct request parameters", t => {
   const { layer } = t.context;
   const { batch } = prepare();
-  layer.setFourBit(true);
 
   const expectedUrl = "url/data/datasets/dataSet/layers/layername/data?token=token2";
   const expectedOptions = {
     data: [
-      { position: [0, 0, 0], zoomStep: 0, cubeSize: 32, fourBit: true },
-      { position: [64, 64, 64], zoomStep: 1, cubeSize: 32, fourBit: true },
+      { position: [0, 0, 0], zoomStep: 0, cubeSize: 32, fourBit: false },
```

Comment:
The unpacking of 4 bit buckets was not tested anyway, right?

Reply:
Nope, wasn't tested anyways.